### PR TITLE
Allow linebreaks in remarks and other improvements to remarks section

### PR DESCRIPTION
**Pull request by victorhuangwq**
## Description

Adds the following

- Better display of scores
- Allow linebreaks
- Categorize/section based on problem type

**Previously:**
image

**Now:**
image

## Motivation and Context

122 remarks that they would prefer line breaks.

## How Has This Been Tested?

Tested locally:
- Tested with negative scores & positive scores
- Tested with and without line breaks
- Tested when there's no annotations
- Tested with multiple problems (and single problem)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have run rubocop for style check. If you haven't, run `overcommit --install && overcommit --sign` to use pre-commit hook for linting
- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: app/views/assessments/viewFeedback.html.erb [hunk 1]
```diff
@@ -22,26 +22,41 @@ Yes, feedback is per problem but we show all annotations for this submission any
...
+          when aFilename > bFilename
+              1
+          else
+              a.line <=> b.line
+          end
+        end
+        s_annotations.each do |annotation| %>
+        <li class="collection-item">
+              <div style='white-space: pre-wrap;'><%= annotation.comment%> </div>
+              <% filename = if annotation.position == -1
```
Comment (damianhxy) [inline app/views/assessments/viewFeedback.html.erb:46]: Should use `get_correct_filename` helper for this

Comment (victorhuangwq) [reply app/views/assessments/viewFeedback.html.erb]: Done

File: app/views/assessments/viewFeedback.html.erb [hunk 2]
```diff
@@ -22,26 +22,35 @@ Yes, feedback is per problem but we show all annotations for this submission any
...
+          else
+              a.line <=> b.line
+          end
+        end
+        s_annotations.each do |annotation| %>
+        <li class="collection-item">
+              <div style='white-space: pre-wrap;'><%= annotation.comment%> </div>
+              <% filename = @get_correct_filename.call(annotation)%>
+              <span> 
+                <b><%= annotation.value? ? "(#{(annotation.value >= 0 ? ' +' : ' -') + annotation.value.abs.to_s})" : "" %></b>
```
Comment (damianhxy) [inline app/views/assessments/viewFeedback.html.erb:48]: Nit: extra space before `+` and `-` (unless that was intended).

Regardless, a suggestion is to simplify the logic with `annotation.value? ? sprintf("(%+.1f)", annotation.value) : ""`. This also adds consistent 1dp rounding behavior (as per speedgrader view) here.

**Review by damianhxy — APPROVED** [review]
Verified that multi-line annotations can now be viewed with newlines.

Left a minor comment, but otherwise lgtm!

File: app/views/assessments/viewFeedback.html.erb (hunk 2 above)
Comment (victorhuangwq) [reply app/views/assessments/viewFeedback.html.erb:48]: thanks, fix taken

**Comment by damianhxy** [comment]
lgtm!

**Pull request merged**